exo-add/src/cli.ls

```diff
@@ -60,6 +62,15 @@ function help
   console.log help-message
 
 
+function check-for-service {service-role, services}
+  existing-services = []
+  for protection-level of services
+    if services[protection-level]
+      existing-services.push Object.keys that
+  if flatten existing-services |> (.includes service-role)
```

Comment:
@kevgo Are you opposed to using `flatten` here? I tried using the 
```
existing-services |> (.values!)
                  |> (.concat!) 
                  |> (.includes service-role)
```
that you had mentioned, but that doesn't seem to work as intended.